create_pr_comments.py

```diff
@@ -0,0 +1,81 @@
+import requests
+import json
+import os
+import logging
+from dotenv import load_dotenv
+from run_gemini import get_review
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(encoding="utf-8", level=logging.DEBUG)
+
+load_dotenv(override=True)
+
+github_api_token = os.environ["GITHUB_API_TOKEN"]
+
+
+def create_pull_request_comment(
+    repository, pull_number, github_api_token, body, commit_id, path, line, side
+):
+
+    pr_url = f"https://api.github.com/repos/{repository}/pulls/{pull_number}/comments"
+    headers = {
+        "Accept": "application/json",
+        "Authorization": f"Bearer {github_api_token}",
+        "X-GitHub-Api-Version": "2022-11-28",
+    }
+    data = {
+        "body": body,
+        "commit_id": commit_id,
+        "path": path,
+        "line": int(line),
+        "side": side,
+    }
+
+    response = requests.post(pr_url, headers=headers, json=data)
+    try:
+        response.raise_for_status()
+    except:
+        logger.exception(response)
+
+    return response.json()
+
+
+def parse_comments(text):
+    comments = []
+    lines = text.splitlines()
+    for line in lines:
+        if line[0] == "{" and line[-1] == "}":
+            try:
+                comment = json.loads(line)
+                if not all(key in comment for key in ["body", "path", "line", "side"]):
+                    raise ValueError("Invalid format: Required key does not exist.")
+                comments.append(comment)
+            except json.JSONDecodeError:
```

Comment:
I would recommend using a dedicated library to parse the JSONL format instead of manually handling it. The `jsonlines` library is specifically designed for this purpose and provides a simple API for reading and writing JSONL files. This improves code readability and maintainability.